### PR TITLE
cert-manager Graduation Due Diligence

### PR DESCRIPTION
**Pull request by kgamanji**
This PR contains the due diligence for cert-manager application to move to graduated status.  

Project application issue: https://github.com/cncf/toc/issues/1306

Overall the project fulfills all the necessary criteria to move to the next level, TAGs assessment and adopter interviews reflect this too. Also, the maintainer's willingness to go through a new process and responsiveness throughout is highly commendable.

### CODE REVIEW

File: proposals/graduation/cert-manager.md [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+- TAG Contributor Strategy recommends to add a process for removing Maintainers (and SC members) for reasons other than inactivity, such as violating the CoC or disruptive behavior.
+- TAG Contributor Strategy recommends to link to the list of official channels and meetings in the contributor docs from the Governance document.
+- TAG Contributor Strategy recommends to gradually build up the sub-projects into their own entities, allowing new contributors to take ownership of them.  This will require adding to the main Governance for these roles.
+- TAG Contributor Strategy recommends to have another Steering meeting, in order to keep Steering members engaged.
+- TAG Contributor Strategy recommends to make sure that the Community repo is linked from appropriate other places, like the main development repos and the contributor docs.
+- TAG Contributor Strategy recommends to figure out a low-effort way to record maintainer decisions for posterity, such as a simple text log.
```
Comment (TheFoxAtWork) [inline proposals/graduation/cert-manager.md:41]: @kgamanji Do you know if the project has open issues to track these to resolution?  If they do, it'd be lovely to have them here so anyone can view their status if they come across the information from the DD.

**Review by TheFoxAtWork — COMMENTED** [review]
LGTM - just one comment about tracking non-blocking improvements.

File: proposals/graduation/cert-manager.md (hunk 1 above)
Comment (kgamanji) [reply proposals/graduation/cert-manager.md:41]: @SgtCoDFish will look into creating issues for the recommendation above

Comment (SgtCoDFish) [reply proposals/graduation/cert-manager.md:41]: In the short term, I've created a tracking issue for this work: https://github.com/cert-manager/community/issues/35

We'll go through and create subtasks linked to that issue where needed, and some of the work is already done (e.g. `govulncheck` / `go vet` is done already and it's just a matter of updating docs really)!

Comment (TheFoxAtWork) [reply proposals/graduation/cert-manager.md:41]: Awesome thanks so much!!

**Review by TheFoxAtWork — APPROVED** [review]
We've a few more days for TOC internal review before opening up public comment here and on the mailing list. LGTM!

**Comment by kgamanji** [comment]
It opened the public comment period that will close on the 20th of September: https://lists.cncf.io/g/cncf-toc/message/8708

**Review by SgtCoDFish — APPROVED** [review]
I thought I'd already left an approving review on this, but I guess I didn't!

Thank you to all involved in this, it's so great to see this all written down. Obviously special thanks to @kgamanji , the fine people of tag-security and tag-contributor-strategy and to the people who were interviewed 😁 

I particular enjoyed reading the interviews - there was a lot of talk of stability which is definitely validating to read because we spend a lot of time working on that!

LGTM!

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Congrats cert-manager team! :D 

I do have 2 small requests before this can be merged^^;;;
@kgamanji if you could, please DCO sign the commits.

Also if you could, please change the filename and directory location for the dd doc.

We've reorganized the directories to keep all project stuff together in one dir (easier to see changes from one level to another etc)

The new location should be here:
https://github.com/cncf/toc/tree/main/projects/cert-manager

**Review by mrbobbytables — APPROVED** [review]
Thanks for fixing the issues :) LGTM

**Pull request merged**